src/client/Microsoft.Identity.Client/Http/HttpManagerFactory.cs

```diff
@@ -14,16 +14,19 @@ namespace Microsoft.Identity.Client.Http
     /// </summary>
     internal sealed class HttpManagerFactory
     {
-        public static IHttpManager GetHttpManager(IMsalHttpClientFactory httpClientFactory, bool withRetry, bool isManagedIdentity)
+        public static IHttpManager GetHttpManager(
+            IMsalHttpClientFactory httpClientFactory,
+            bool withRetry,
```

Comment:
Can you get rid of these 2 flags? Just reading the code is tricky here. Might as well have the upper layer inject the retry policy directly.

Reply:
Discussed offline, these flags cannot be eliminated but are needed to use the correct policy